mscore/preferenceslistwidget.cpp

```diff
@@ -401,14 +398,13 @@ void DoublePreferenceItem::apply()
                   }
             else if (_editorComboBox) {
                   double newValue = _editorComboBox->currentData().toDouble();
-                  _initialValue = newValue;
                   PreferenceItem::apply(newValue);
+                   _initialEditorIndex = _editorComboBox->currentIndex();
```

Comment:
this here is where the merge conflict happens and only due to an excess 'space'

Reply:
I'll take care of it when I am back at my computer.